src/test/resources/ResourcesForFeatureFiles/SprintTestCases.feature

```diff
@@ -38,8 +38,8 @@ Feature: Verify
     When  click Search Box and type 'Bora'
     And   select 'Bora Bora'
     And   I click Check In
-    Then  I select Check-in date as '1'
-    Then  I select Check-out date from calendar as '15'
+    Then  I select Check-in date as '4'
+    Then  I select Check-out date from calendar as '20'
```

Comment:
Looks like above 2 lines are action steps and should be labelled as **And** and same observation for other Action steps.